CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [1.12.0] - 2021-04-29
+### Features
```

Comment:
```suggestion
### Added
```

We use https://keepachangelog.com/en/1.0.0/  format to create changelog entries.

Reply:
I saw this pattern, but our CHANGELOG.md is using "Features" instead of "Added". How do you suggest that I do? @gris

Reply:
This project was forked, so we don't control the history before our changes. I suggest adhering to keepachangelog format.

Reply:
I understand. So ok.